sys/fs/fuse/fuse_vnops.c

```diff
@@ -1474,7 +1474,8 @@ fuse_vnop_lookup(struct vop_lookup_args *ap)
 			 * Since the file system doesn't support ".." lookups,
 			 * we have no way to find this entry.
 			 */
-			return ESTALE;
+			return (EXTERROR(ESTALE, "This daemon does not support "
```

Comment:
I think "server" is the preferred language here.
```suggestion
			return (EXTERROR(ESTALE, "This server does not support "
```